paramiko/channel.py

```diff
@@ -851,10 +849,10 @@ def sendall_stderr(self, s):
         """
         Send data to the channel's "stderr" stream, without allowing partial
         results.  Unlike `send_stderr`, this method continues to send data
-        from the given string until all data has been sent or an error occurs.
-        Nothing is returned.
+        from the given bytestring until all data has been sent or an error
+        occurs. Nothing is returned.
 
-        :param str s: data to send to the client as "stderr" output.
+        :param bytes s: data to send to the client as "stderr" output.
```

Comment:
This type was changed to `bytes` here -- was the analogous type on line 831 in the docstring for `sendall()` also supposed to be changed to `bytes`? (I can't tag a comment on a line that isn't diffed, annoyingly.)

Reply:
Yea, that one was missed for some reason, good spot!